build/valgrind/openssl.supp

```diff
@@ -29,3 +29,19 @@
    fun:_ZN5Catch8runTestsERKNS_3PtrINS_6ConfigEEE
    fun:main
 }
+{
+   <same-as-above-but-for-another-version-of-libevent>
+   Memcheck:Addr4
+   fun:tls_check_name
+   fun:_ZNSt17_Function_handlerIFvN2mk5ErrorEP11buffereventEZNS0_3net11connect_sslES3_P6ssl_stSsSt8functionIS4_ENS0_3VarINS0_7ReactorEEENSA_INS0_6LoggerEEEEUlS1_S3_E_E9_M_invokeERKSt9_Any_dataOS1_OS3_
+   fun:mk_bufferevent_on_event
+   fun:bufferevent_run_deferred_callbacks_locked
+   fun:event_process_active_single_queue
```

Comment:
This function above is not part of libevent-2.0, so we need two suppression files.